posthog/tasks/usage_report.py

```diff
@@ -1219,6 +1222,28 @@ def _get_full_org_usage_report_as_dict(full_report: FullUsageReport) -> dict[str
     return dataclasses.asdict(full_report)
 
 
+def _queue_report(producer, organization_id: str, full_report_dict: dict[str, Any]) -> None:
+    logger.info(f"Sending usage report for organization {organization_id}")  # noqa T201
+
+    json_data = json.dumps(
+        {"organization_id": organization_id, "usage_report": full_report_dict}, separators=(",", ":")
+    )
+    compressed_bytes = gzip.compress(json_data.encode("utf-8"))
+    compressed_b64 = base64.b64encode(compressed_bytes).decode("ascii")
+
+    message_attributes = {
+        "content_encoding": {"DataType": "String", "StringValue": "gzip"},
```

Comment:
looks like this stuff is all generated automatically in the producer

Reply:
https://github.com/PostHog/posthog/blob/0ce77a38b74a95c370bd1069037e206ac68fae02/ee/sqs/SQSProducer.py#L174

Reply:
No wonder you can ship to prod, your like a machine analyzing the code.

Fixed and tested:
```
>>> message_attributes = {
...     "content_encoding": "gzip",
...     "content_type": "application/json",
... }
>>> SQSProducer._format_message_attributes({}, message_attributes)
{'content_encoding': {'DataType': 'String', 'StringValue': 'gzip'}, 'content_type': {'DataType': 'String', 'StringValue': 'application/json'}}
```